examples/foomedical/.github/workflows/build.yml

```diff
@@ -0,0 +1,21 @@
+name: Build
```

Comment:
Do we still want these`.github/workflows` files?

Reply:
Likely not, we should have it build in the main repo, otherwise we'd need to make changes in the listening repo which we are trying to avoid.